command/v7/push_command_test.go

```diff
@@ -1290,5 +1292,23 @@ var _ = Describe("push Command", func() {
 					"--task", "--strategy=canary",
 				},
 			}),
+
+		Entry("max-in-flight is passed without strategy",
+			func() {
+				cmd.MaxInFlight = 10
+			},
+			translatableerror.RequiredFlagsError{
```

Comment:
[question, non-blocking] does it feel to you/anyone else like we are over-testing? I feel like I've seen several of the same error expectations at different levels
edit: nevermind if this is the same test for the different commands

Reply:
We do this test for all the commands we changed

Reply:
sounds good, thanks